generated/nidaqmx/constants.py

```diff
@@ -293,12 +297,12 @@ class EveryNSamplesEventType(Enum):
 
 
 class ExcitationDCorAC(Enum):
-    USE_DC = 10050  #: DC excitation.
-    USE_AC = 10045  #: AC excitation.
+    DC = 10050  #: DC excitation.
+    AC = 10045  #: AC excitation.
```

Comment:
This is fine

Reply:
Updated enum values as in expected constants.py